mbuild/formats/gomc_conf_writer.py

```diff
@@ -2012,30 +2120,159 @@ def __init__(
         else:
             self.input_error = True
             print_error_message = (
-                "ERROR: The ensemble type selection of {}  is not a valid ensemble option. "
-                "Please choose the 'NPT', 'NVT', 'GEMC_NVT','GEMC_NPT', or 'GCMC' "
+                "ERROR: The ensemble type selection of '{}' is not a valid ensemble option. "
+                "Please choose the 'NPT', 'NVT', 'GEMC_NVT', 'GEMC_NPT', or 'GCMC' "
                 "ensembles".format(ensemble_type)
             )
             raise ValueError(print_error_message)
 
+        # check if check_input_files_exist is a boolean
+        if check_input_files_exist is not None:
+            _check_if_bool("check_input_files_exist", check_input_files_exist)
+
+        # set the restart attributes:
+        if Restart is not None:
```

Comment:
```suggestion
        if Restart:
```
This behaves the same.

Reply:
changed